lib/datadog/profiling/tasks/help.rb

```diff
@@ -0,0 +1,17 @@
+# typed: true
+module Datadog
+  module Profiling
+    module Tasks
+      # Prints help message for usage of `ddtrace`
+      class Help
+        def run
+          puts %(
+  Usage: ddtrace [command] [arguments]
```

Comment:
Oops, you did not change this spotted that it was wrong :)

```suggestion
  Usage: ddtracerb [command] [arguments]
```